libs/coin-modules/coin-canton/README.md

```diff
@@ -6,6 +6,50 @@ This repository contains the Coin Canton module for Ledger Live.
 
 The Coin Canton module provides support for the Canton blockchain within the Ledger Live application
 
-## Test setup
+## Protobuf Generation
 
-Create .env.integ.test file based on .env.integ.test.example
+The Canton module uses Protocol Buffers (protobuf) for communication with the Ledger device. The protobuf definitions are automatically generated from external repositories.
+
+### When to use
+
+When new portobuf applied to [gateway](https://github.com/LedgerHQ/canton-protos-scala) and [app-canton](https://github.com/LedgerHQ/app-canton) use the `generate-proto` script to update TypeScript bindings.
+
+### How to use
+
+```bash
+# Navigate to the coin-canton directory
+cd libs/coin-modules/coin-canton
+
+# Run the protobuf generation script
+pnpm generate-proto
+```
+
+### Description
+
+The script performs the following operations:
```

Comment:
can you explain the full flow in documentation ? where / why those files are used / what it contains, not just how to generate them